centos-5.11-i386.json

```diff
@@ -102,21 +102,18 @@
   ],
   "post-processors": [
     {
-      "output": "builds/{{.Provider}}/opscode_centos-5.11-i386_chef-{{user `chef_version`}}.box",
+      "output": "builds/{{.Provider}}/opscode_centos-5.11-i386.box",
```

Comment:
Ah, that's a good catch. I'm hoping to update kitchen-vagrant just as soon as we can get boxes publishing up to Atlas. That could mean in the very short term that the box would have to be renamed when uploading to the S3 bucket. About 2 PRs down the road, I'm hoping to change the box file names even more so this is going to be important to keep in mind.